lib/sles4sap/azure_cli.pm

```diff
@@ -1661,3 +1664,118 @@ sub az_storage_blob_update {
 
     return script_run(join(' ', @az_cmd));
 }
+
+=head2 az_keyvault_list
+
+    az_keyvault_list(resource_group=>'resource group' [, query=>'[].id']);
+
+Lists all keyvault resource names located in specified resource group.
+Output can be modified using B<query> argument.
+
+=over
```

Comment:
other functions in this file have the number of  arguments here.  I don't care - just an observation

Reply:
Oh yes... that is what we found out with Michele lately. 
The number after `=over` is not number of arguments but an offset. This offset is however not even considered by our perlpod formatter so it does basically nothing. I just removed the remaining number in other function.
https://perldoc.perl.org/perlpod#=over-indentlevel

```
"=over" indicates how far over to indent, generally in ems (where one em is the width of an "M" in the document's base font) or roughly comparable units; if there is no indentlevel option, it defaults to four. **(And some formatters may just ignore whatever indentlevel you provide.)** In the stuff in =item stuff..., you may use formatting codes, as seen here:
```